extensions/amp-sidebar/1.0/sidebar.js

```diff
@@ -109,6 +111,26 @@ function SidebarWithRef(
     setMounted
   );
 
+  useEffect(() => {
+    const sidebarElement = sidebarRef.current;
+    const backdropElement = backdropRef.current;
+    if (!sidebarElement || !sidebarElement.ownerDocument || !backdropElement) {
+      return;
+    }
+
+    const document = sidebarElement.ownerDocument;
+    const keydownCallback = (event) => {
+      if (event.key == Keys.ESCAPE) {
+        event.preventDefault();
```

Comment:
FMI: Is there a case you had in mind that you are trying to prevent with this call? It's not obvious to me why it is necessary.

Reply:
Nothing specifically in mind, just carried over from the port of `0.1`.  Although, I suppose it helps in that hypothetically another listener can check `event.defaultPrevented` on this event.

Reply:
`preventDefault` might be helpful, but what we definitely need is `event.stopImmediatePropagation()`.